.eslintrc

```diff
@@ -29,6 +29,8 @@
 
     "@typescript-eslint/camelcase": "off", // off as dns-packet has a lot of stuff like this
     "@typescript-eslint/no-non-null-assertion": "off",
-    "@typescript-eslint/no-use-before-define": ["error", {"classes": false}]
+    "@typescript-eslint/no-use-before-define": ["error", {"classes": false}],
+
+    "@typescript-eslint/no-unsafe-declaration-merging": "off"
```

Comment:
What is the reason we disable that? Looking at the documentation of that rule it sounds like something we would like to keep enabled?
Otherwise, can we add a small documentation comment to explain why we disabled this (similar to the one above for `camelcase`).

Reply:
```
/ciao/src/CiaoService.ts
  203:26  error  Unsafe declaration merging between classes and interfaces  @typescript-eslint/no-unsafe-declaration-merging
  276:14  error  Unsafe declaration merging between classes and interfaces  @typescript-eslint/no-unsafe-declaration-merging

/ciao/src/NetworkManager.ts
   83:26  error  Unsafe declaration merging between classes and interfaces  @typescript-eslint/no-unsafe-declaration-merging
  102:14  error  Unsafe declaration merging between classes and interfaces  @typescript-eslint/no-unsafe-declaration-merging

/ciao/src/responder/TruncatedQuery.ts
  15:26  error  Unsafe declaration merging between classes and interfaces  @typescript-eslint/no-unsafe-declaration-merging
  23:14  error  Unsafe declaration merging between classes and interfaces  @typescript-eslint/no-unsafe-declaration-merging
```

I'm happy to add a - to each of these files instead? Might make more sense actually, so future cases get highlighted and can be avoided.

```
/* eslint-disable @typescript-eslint/no-unsafe-declaration-merging */
```

Reply:
Ah, interesting. We use that mechanisms to provide typing for event handlers via nodejs `EventEmitter`. Might make sense to disable this rules on a per case basis, for each individual line?

Would something like `// eslint-disable-next-line @typescript-eslint/no-unsafe-declaration-merging` (I think that's the syntax for that) work for these cases?